src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const selectTab = id => {
+    if (id !== activeTab.id) {
+      onTabSelected(id);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title }) => (
+            <li
+              key={id}
+              className={cn({ 'is-active': id === activeTab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${id}`}
+                data-cy="TabLink"
+                onClick={() => selectTab(id)}
+              >
+                {title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {activeTab.content}
```

Comment:
I recommend adding a check for activeTab. Like this:
`{activeTab ? activeTab.content : ''}`